ydb/core/kqp/ut/federated_query/generic/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.4'
```

Comment:
We already have a configured file here: https://github.com/ydb-platform/ydb/blob/main/ydb/library/yql/providers/generic/connector/tests/docker-compose.yml

Let's make a reusable recipe from it and use it in both places

The example of making a recipe: https://github.com/ydb-platform/ydb/tree/main/ydb/tests/tools/s3_recipe

Reply:
> We already have a configured file here: https://github.com/ydb-platform/ydb/blob/main/ydb/library/yql/providers/generic/connector/tests/docker-compose.yml

> Let's make a reusable recipe from it and use it in both places

This `docker-compose.yml` will highly likely split into several parts soon (integration tests will be separated into different directories as well). I'm afraid this will make it impossible to reuse this file in both kinds of tests.